torch/onnx/onnx_supported_ops.py

```diff
@@ -0,0 +1,110 @@
+from inspect import signature, _empty  # type: ignore[attr-defined]
```

Comment:
can you make everything except the `onnx_supported_ops` function private so users don't depend on it?